docs/docker-machine-drivers.md

```diff
@@ -55,6 +55,18 @@ $ newgrp libvirt
 <a name="xhyve-driver"></a>
 ## xhyve driver
 
+Minishift is currently tested against `docker-machine-driver-xhyve` 0.3.1
```

Comment:
@budhrg I thinks when you specify `0.3.1` that means referring to version. Also as @Preeticp said there is another issue which will take care of all version info against minishift and more changes in documentation.